assets/prometheus-k8s/rules.yaml

```diff
@@ -522,14 +522,6 @@ spec:
       record: build_error_rate
   - name: kubernetes-absent
     rules:
-    - alert: AlertmanagerDown
```

Comment:
This doesn’t look right. I think we accidentally didn’t append the telemetry Job but instead overwrote the whole array.

Reply:
This is what I was thinking. I’ll fix this in the telemeter repo